graylog2-web-interface/src/components/field-analyzers/QuickValuesOptionsForm.jsx

```diff
@@ -0,0 +1,114 @@
+    onSave: PropTypes.func.isRequired,
+    onCancel: PropTypes.func.isRequired,
+  },
+
+  getInitialState() {
+    return {
+      limit: this.props.limit,
+      tableSize: this.props.tableSize,
+      order: this.props.order,
+      stackedFields: this.props.stackedFields,
+    };
+  },
+
+  _changeConfig(key, value) {
+    const state = _.cloneDeep(this.state);
+    state[key] = value;
+    this.setState(state);
+  },
+
+  _onChange(event) {
+    this._changeConfig(event.target.name, FormsUtils.getValueFromInput(event.target));
+  },
+
+  _onStackedFieldChange(values) {
+    this._changeConfig('stackedFields', values);
+  },
+
+  _onCancel(event) {
+    event.preventDefault();
+    this.props.onCancel();
+  },
+
+  _onSave(e) {
+    e.preventDefault();
+    this.props.onSave(this.state);
+  },
+
+  render() {
+    const fieldOptions = this.props.stackedFieldsOptions
+      .filter(field => !field.name.startsWith('gl2_')) // Do not include Graylog internal fields
+      .filter(field => field.name !== this.props.field) // Do not include the main QuickValues field
+      .map((field) => {
+        return { value: field.name, label: field.name };
+      });
+
+    return (
+      <Row>
+        <Col md={6}>
+          <form className="form" onSubmit={this._onSave}>
+            <fieldset style={{ paddingLeft: 15 }}>
+              <Input type="number"
+                     id="limit"
+                     name="limit"
+                     label="Number of top/bottom values"
+                     autoFocus
+                     required
+                     onChange={this._onChange}
+                     value={this.state.limit} />
+              <Input type="number"
+                     id="tableSize"
+                     name="tableSize"
+                     label="Total table size"
+                     required
+                     onChange={this._onChange}
+                     value={this.state.tableSize} />
+              <Input label="Sort options">
+                <Input type="radio"
+                       name="order"
+                       label="Top values"
+                       checked={this.state.order === 'desc'}
+                       value="desc"
+                       onChange={this._onChange} />
+                <Input type="radio"
+                       name="order"
+                       label="Bottom values"
+                       checked={this.state.order === 'asc'}
+                       value="asc"
+                       onChange={this._onChange} />
+              </Input>
+
+              <Input label="Stacked fields">
+                <MultiSelect options={fieldOptions}
+                             value={this.state.stackedFields}
+                             onChange={this._onStackedFieldChange} />
+              </Input>
+
+              <Button type="submit" bsStyle="success" bsSize="small">Update</Button>
+              {' '}
```

Comment:
We should use `<ButtonToolbar>` for this. From react-bootstrap docs:
>Because React doesn't output newlines between elements, buttons on the same line are displayed flush against each other. To preserve the spacing between multiple inline buttons, wrap your button group in <ButtonToolbar />.